lib/services/android-project-service.ts

```diff
@@ -522,81 +404,14 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		}
 	}
 
-	private getTargetFromAndroidManifest(configurationFilePath: string): string {
-		let versionInManifest: string;
-		if (this.$fs.exists(configurationFilePath)) {
-			const targetFromAndroidManifest: string = this.$fs.readText(configurationFilePath);
-			if (targetFromAndroidManifest) {
-				const match = targetFromAndroidManifest.match(/.*?android:targetSdkVersion=\"(.*?)\"/);
-				if (match && match[1]) {
-					versionInManifest = match[1];
-				}
-			}
-		}
-
-		return versionInManifest;
-	}
-
-	private async executeCommand(opts: { projectRoot: string, gradleArgs: any, childProcessOpts?: SpawnOptions, spawnFromEventOptions?: ISpawnFromEventOptions, message: string }): Promise<ISpawnResult> {
-		if (this.$androidToolsInfo.getToolsInfo().androidHomeEnvVar) {
-			const { projectRoot, gradleArgs, message, spawnFromEventOptions } = opts;
-			const gradlew = this.$hostInfo.isWindows ? "gradlew.bat" : "./gradlew";
-
-			if (this.$logger.getLevel() === "INFO") {
-				gradleArgs.push("--quiet");
-			}
-
-			this.$logger.info(message);
-
-			const childProcessOpts = opts.childProcessOpts || {};
-			childProcessOpts.cwd = childProcessOpts.cwd || projectRoot;
-			childProcessOpts.stdio = childProcessOpts.stdio || "inherit";
-			let commandResult;
-			try {
-				commandResult = await this.spawn(gradlew,
-					gradleArgs,
-					childProcessOpts,
-					spawnFromEventOptions);
-			} catch (err) {
-				this.$errors.failWithoutHelp(err.message);
-			}
-
-			return commandResult;
-		}
-	}
-
-	private isAndroidStudioCompatibleTemplate(projectData: IProjectData, frameworkVersion?: string): boolean {
-		const currentPlatformData: IDictionary<any> = this.$projectDataService.getNSValue(projectData.projectDir, constants.TNS_ANDROID_RUNTIME_NAME);
-		const platformVersion = (currentPlatformData && currentPlatformData[constants.VERSION_STRING]) || frameworkVersion;
-
-		if (!platformVersion) {
-			return true;
-		}
-
-		if (platformVersion === constants.PackageVersion.NEXT || platformVersion === constants.PackageVersion.LATEST || platformVersion === constants.PackageVersion.RC) {
-			return true;
-		}
-
-		const androidStudioCompatibleTemplate = "3.4.0";
-		const normalizedPlatformVersion = `${semver.major(platformVersion)}.${semver.minor(platformVersion)}.0`;
-
-		return semver.gte(normalizedPlatformVersion, androidStudioCompatibleTemplate);
-	}
-
 	private getLegacyAppResourcesDestinationDirPath(projectData: IProjectData): string {
```

Comment:
Maybe we can remove the support for legacy App_Resources structure for 6.0

Reply:
We discussed this and decided to keep the support for legacy structure for `6.0.0` release